### PR TITLE
add help keyword response

### PR DESCRIPTION
**Pull request by abrahamsangha**
#### Why did you need to make these changes?

* this pr enables users to type 'help' and receive info on how to use swifty-bot

#### What changes did you make?

* added a route, added a spec for that route, added a constant assigned to the help verbage

#### Links to documentation of new tech:

* rtfm

#### Obligatory gif

bald_eagle

### CODE REVIEW

**Comment by alanbsmith** [comment]
Looks good to me, @abrahamsangha :shipit:

File: spec/help.spec.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+var request = require('request');
+var helpers = require('./helpers');
+require('dotenv').config();
+
+describe('Help', function() {
+  beforeEach(helpers.setupWebserver);
+
+  afterEach(helpers.teardownWebserver);
+
+  it('returns status code 200', function(done) {
```
Comment (craftninja) [inline spec/help.spec.js]: Maybe add a second test that it gives a 404 without a token (example in pairs)?

File: routes/help.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+HelpOutput += "*pairs* -- _allows you to register with whom you would like to pair_\n";
+HelpOutput += "*pairing* -- _allows you to register with whom you would like to pair_\n";
+HelpOutput += "*list* -- _lists all pairs from today_\n";
+
+
+router.post('/', (req, res, next) => {
+  if (req.body.token === process.env.HELP_WEBHOOK_TOKEN){
+    var swifty = req.app.get('swifty');
+    swifty.postToChannel('swiftly', HelpOutput);
+    res.send({status: "ok"});
```
Comment (craftninja) [inline routes/help.js]: Was `res.send` as opposed to `res.json` intentional?

File: spec/help.spec.js (hunk 1 above)
Comment (abrahamsangha) [inline spec/help.spec.js]: It seems to me we'd actually want a 401 if no token is present: https://tools.ietf.org/html/rfc7235#section-3.1
Thoughts?

File: routes/help.js (hunk 2 above)
Comment (abrahamsangha) [inline routes/help.js]: No. Is there a reason to change it?

File: spec/help.spec.js (hunk 1 above)
Comment (craftninja) [inline spec/help.spec.js]: I'm from the school of thought that is someone is trying to access something they "shouldn't", you 404 them so they don't know the resource is there. There are already 404s set up in other routes for attempted access without a token, so if we go the 401 way, we should probably do it in a refactoring and hit them all at the same time. I'm down to discuss.

File: routes/help.js (hunk 2 above)
Comment (alanbsmith) [inline routes/help.js]: The format doesn't matter to the server per se. We need to send something back to keep it from blowing up. Style consistency is another thing to consider though. Not sure how this is being done in the rest of the app.

Comment (craftninja) [inline routes/help.js]: `res.send` sends text, `res.json` sends a json object, and the rest of our responses return json.

**Pull request merged**